addons/packages/app-toolkit/0.2.0/bundle/config/developer-namespace.yaml

```diff
@@ -0,0 +1,48 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ if/end data.values.developer_namespace != "default":
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: #@ data.values.developer_namespace
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-credentials
+  namespace: #@ data.values.developer_namespace
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: e30K
+
+---
+apiVersion: v1 
+kind: ServiceAccount
+metadata:
+  name: default
+  namespace: #@ data.values.developer_namespace
+  annotations:
+    kapp.k14s.io/create-strategy: fallback-on-update
+secrets:
+- name: registry-credentials
+imagePullSecrets:
+- name: registry-credentials
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ootb-supply-chain-source-to-url-workload
+  namespace: #@ data.values.developer_namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: ootb-supply-chain-source-to-url-workload
```

Comment:
referencing a specific cluster role for a specific supply chain is fragile. Using an aggregated cluster role to collect permissions into a stable name that can then be bound is preferable.

Reply:
Notes from Zoom Call:
This could make it onerous if the user adds another supply chain because it means we'll have to specify another role binding. Aggregating it would mean less toil in setting up other supplychains because we could use the same role and binding and let the aggregated role fetch other roles by using labels.

Nice to have feature that we'll add to the backlog